coffea/processor/executor.py

```diff
@@ -1093,6 +1120,7 @@ def run_uproot_job(fileset,
     mmap = executor_args.pop('mmap', False)
     schema = executor_args.pop('schema', None)
     nano = executor_args.pop('nano', False)
+    use_dataframes = executor_args.pop('use_dataframes', False)
```

Comment:
Can you add a small section to the docstring describing this feature? Also this would be a good spot to check if the executor is `dask_executor` since that is the only executor that supports this option

Reply:
Sure!